src/SimpleSoftwareIO/SMS/Drivers/NexmoSMS.php

```diff
@@ -68,7 +68,81 @@ public function send(OutgoingMessage $message)
         $this->buildCall('/sms/json');
         $this->buildBody($data);
 
-        return $this->postRequest();
+        $response = $this->postRequest();
+        $body = json_decode($response->getBody(), true);
+        if ($this->hasError($body)) {
+            $this->handleError($body);
+        }
+        
+        return $response;
+    }
+    
+    /**
+     * Checks if the transaction has an error
+     *
+     * @param $body
+     * @return bool
+     */
+    protected function hasError($body)
+    {
+        if ($this->hasAResponseMessage($body) && $this->hasProperty($this->getFirstMessage($body), 'status')) {
+            $firstMessage = $this->getFirstMessage($body);
+            return (int)$firstMessage['status'] !== 0;
+        }
+        return false;
+    }
+    
+    /**
+     * Log the error message which ocurred
+     *
+     * @param $body
+     */
+    protected function handleError($body)
+    {
+        $firstMessage = $this->getFirstMessage($body);
+        $error = 'An error occurred. Nexmo status code: ' . $firstMessage['status'];
+        if ($this->hasProperty($firstMessage, 'error-text')) {
+            $error = $firstMessage['error-text'];
+        }
+        
+        \Log::error($error);
```

Comment:
I'm not sure if we should have this log.  This might not be available on an installation that is not using Laravel.